File: public/nootan-20160520092127/C/little-sisters.c
// Name : Little Sisters
#include <stdio.h>
int main() {
	// input:
	// n : number of sisters Hikki has
	int n, solution[100][100], idx, jdx;
	char tmp[100];
	scanf("%d", &n);


	// write your code here
	// store your results in `solution`

	// output
	for(idx = 0; idx< n; idx++) {
		for(jdx = 0;jdx<n;jdx++) {
			printf("%d ",solution[idx][jdx]);
		}
		printf("\n");
	}
	return 0;
}

File: public/nootan-20160520092127/JAVA/little-sisters.java
// Name : Little Sisters
import java.util.*;
import java.lang.*;
import java.io.*;
class Main {
	public static void main (String[] args) throws java.lang.Exception {
		// input:
		// a : size
		// b : size
		// x : ratio
		// y : ratio
				Scanner in = new Scanner(System.in);
		String[] elements = (in.nextLine()).split(" ");
		int a = Integer.parseInt(elements[0]);
		int b = Integer.parseInt(elements[1]);
		int x = Integer.parseInt(elements[2]);
		int y = Integer.parseInt(elements[3]);


		// write your code here
		// store your results in `new_size_x`,`new_size_y`

		// output
				System.out.println(new_size_x+" "+new_size_y);

	}
}